full_bear/tests/test_api_v1/test_views/test_user.py

```
@@ -0,0 +1,31 @@
+import pytest
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from full_bear.models import UserModel
+
+
+@pytest.mark.django_db()
+def test_register_new_user_successfully(api_client: APIClient) -> None:
+    EXPECTED_USERS_NUM_BEFORE_REGISTRATION = 0
+    EXPECTED_USERS_NUM_AFTER_REGISTRATION = 1
+
+    # GIVEN
+    users = UserModel.objects.all()
+    users_num_before_registration = users.count()
```

There should be check for count of users before registration. We want to use `fail fast`, if the number of users doesn't math `EXPECTED_USERS_NUM_BEFORE_REGISTRATION` in this place, we don't want to run whole tests.

So please add `assert users_num_before_registration == EXPECTED_USERS_NUM_BEFORE_REGISTRATION` here, and remove it from `# THEN`